src/main/deploy/WitzMech.xml

```diff
@@ -1,18 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<mechanism xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+<mechanism xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns="http://team302.org/robot">
```

Comment:
Why are we putting this in the deploy folder?  It looks like a section of robot.xml that I thought we were eliminating.